src/python/AsyncStageOut/TransferDaemon.py

```diff
@@ -227,7 +227,7 @@ def oracleSiteUser(self, db):
 
             self.doc_acq = str(result)
             for i in range(len(user)):
-                if not user[i]:
+                if not user[i] or user[i] in [None, 'NULL']:
```

Comment:
None should already be catched by the first clause of the if. Is it necessary? Did you mean `'None'`?